src/utils/grid.css

```diff
@@ -1,3 +1,201 @@
+.u-1\/12 {
```

Comment:
@wesoudshoorn Do we want to prefix this utility with something like: `.u-grid-1/12`?

Reply:
That seems reasonable. Would we want to drop the `/12` postfix?

`.u-grid-1`, `.u-grid-2` etc?

Reply:
For me personally it helps to see how many cols something spans. But I don't have a strong opinion about it.

Reply:
Updated it to include the `-grid-` part in the name. For now I kept the /12. If you prefer we can remove it, but I don't mind having it in.